tests/Findologic/Request/NavigationRequestFactoryTest.php

```diff
@@ -71,6 +74,8 @@ public function testNavigationRequestInstance(): void
         $this->assertSame($expectedIpAddress, $params['userip']);
         $this->assertSame($expectedAdapter, $params['outputAdapter']);
         $this->assertSame($expectedHost, $params['shopurl']);
+        $this->assertSame($expectedShopType, $params['shopType']);
+        $this->assertSame($expectedShopVersion, $params['shopVersion']);
```

Comment:
Why are these asserted before the `cat`? To have the same structure at the top as at the assertions down here, either move these two assertions after the `cat` assertion, or move the declarations at the top above the `$expectedCategoryPath` declaration.

Reply:
What I have done is, I have declared '$expectedShopType' and 'expectedShopVersion' variables after '$expectedCategoryPath', and the same structure goes for assertions